R/install.r

```diff
@@ -96,9 +96,20 @@ install <-
 
   # If building vignettes, make sure we have all suggested packages too.
   if (build_vignettes && missing(dependencies)) {
-    dependencies <- TRUE
+    dependencies <- standardise_dep(TRUE)
+  } else {
+    dependencies <- standardise_dep(dependencies)
   }
-  install_deps(pkg, dependencies = dependencies, upgrade = upgrade_dependencies,
+
+  initial_deps <- dependencies[dependencies != "Suggests"]
+  final_deps <- dependencies[dependencies == "Suggests"]
```

Comment:
I think Enhances should be fine to installed in the initial installation. Note `Enhances:` are not installed unless explicitly set by default (see ?install.packages).
> logical indicating whether to also install uninstalled
packages which these packages depend on/link
to/import/suggest (and so on recursively).  Not used if
‘repos = NULL’.  Can also be a character vector, a subset of
‘c("Depends", "Imports", "LinkingTo", "Suggests",
  "Enhances")’
          
> Only supported if ‘lib’ is of length one (or missing), so it is
unambiguous where to install the dependent packages.  If this is not the case
it is ignored, with a warning.

>  The default, ‘NA’, means ‘c("Depends", "Imports",
  "LinkingTo")’.

> ‘TRUE’ means to use ‘c("Depends", "Imports", "LinkingTo",
  "Suggests")’ for ‘pkgs’ and ‘c("Depends", "Imports",
  "LinkingTo")’ for added dependencies: this installs all the
packages needed to run ‘pkgs’, their examples, tests and
vignettes (if the package author specified them correctly).

> In all of these, ‘"LinkingTo"’ is omitted for binary
packages.